sdk/monitor/monitor-query-logs/test.env

```diff
@@ -0,0 +1,24 @@
+# Retrieve these values from an instance in the Azure Portal.
+MONITOR_WORKSPACE_ID=<The "Workspace ID", as given in the Azure portal for your Log Analytics resource>
+METRICS_RESOURCE_ID=<An Azure resource ID to query for metrics. Ex: /subscriptions/<full resource path>>
+
+# Used by tests and samples to populate data
+APPLICATIONINSIGHTS_CONNECTION_STRING=<Azure Application Insights connection string>
```

Comment:
Let's make sure all env vars that are actually used be listed here and nothing else.

Reply:
updated